packages/ui-alerts/src/Alert/index.tsx

```diff
@@ -166,7 +167,7 @@ class Alert extends Component<AlertProps> {
     const liveRegion = this.getLiveRegion()
     if (liveRegion) {
       // @ts-expect-error ts-migrate(2339) FIXME: Property 'srid' does not exist on type 'Alert'.
-      this.srid = uid('Alert')
+      this.srid = hashInstance('Alert', this.props.instanceMapCounter)
```

Comment:
If we do that, will be the `instanceMapCounter` being shown to the end user as a suggestion?

Reply:
It will, but so do `styles` and `makeStyles` too. But that's why I put 2 warnings in withStyle as well to warn ppl that it shouldn't be used directly :) https://github.com/instructure/instructure-ui/blob/a2cfa0cd8bf43d6072f1b352bbd6e43542081be8/packages/emotion/src/withStyle.tsx#L167

Reply:
Thanks, done pretty much the same